modules/document_repository/php/document_repository.class.inc

```diff
@@ -78,10 +78,9 @@ class Document_Repository extends \NDB_Menu_Filter
                           'v.Data_dir',
                           'v.Date_uploaded',
                          );
-
-        $openAccordion = $_GET['openAccordion'];
+        $openAccordion = isset($_GET['openAccordion']) ?? '';
```

Comment:
This idiom doesn't make sense. 

isset returns a bool true or false, never null, so the `?? ''` null coalesce can never be reached, and even if it could be reached, it wouldn't really make sense for the possible values of the variable to be either true, false, or the empty string.